neo/Consensus/ConsensusContext.cs

```diff
@@ -94,7 +94,10 @@ public void Deserialize(BinaryReader reader)
                     transactions[i] = Transaction.DeserializeFrom(reader);
                 Transactions = transactions.ToDictionary(p => p.Hash);
             }
-            Preparations = reader.ReadVarBytes().Select(p => p > 0).ToArray();
+            Preparations = reader.ReadSerializableArray<UInt256>();
+            for (int i = 0; i < Preparations.Length; i++)
+                if (Preparations[i].Equals(UInt256.Zero))
```

Comment:
`Preparations[i]!=null && Preparations[i].Equals`

Reply:
https://github.com/neo-project/neo/blob/9ce07a5c22e0d275fdc5c7004320595533874e73/neo/Consensus/ConsensusContext.cs#L258-L262

It is impossiable for `Preparations[i]` to be `null`.